weave/trace/weave_client.py

```diff
@@ -693,6 +703,9 @@ def finish_call(
         *,
         postprocess_output: Optional[Callable[..., Any]] = None,
     ) -> None:
+        if should_convert_paths_to_images():
+            output = convert_paths_to_images(output)
+        self._save_nested_objects(output)
```

Comment:
are you doing this to trigger img saving?

Reply:
yeah, if the setting is True, we rip through the entire output and replace any image paths with real images.

Reply:
I meant the `_save_nested_objects` part

Reply:
oh, yes, we do the conversion first so that the actual image (bytes) get saved instead of the url. This path uses grabs the `ImagePath` serializer to save it